smbus2/smbus2.py

```diff
@@ -316,6 +317,18 @@ def close(self):
             os.close(self.fd)
             self.fd = None
 
+    def enable_pec(self, enable=True):
+        """
+        Enable/Disable PEC (Packet Error Checking) - SMBus 1.1 and later
+        
```

Comment:
Here you need to document the input parameter(s). Please see documentation elsewhere in the code for reference.